package-lock.json

```diff
@@ -912,7 +912,6 @@
       "integrity": "sha512-UlLAnTPrFdNGoFtbSXwcGFQBtQZJCNjaN6hQNP3UPvuNXT1i82N26KL3dZeIpNalWywr9IuQuncaAfUaS1g6sQ==",
       "dev": true,
       "license": "MIT",
-      "peer": true,
```

Comment:
The package-lock.json probably changed because the team are using different npm versions locally - we should align to the same across the team